### DISCUSSION TITLE
How to handle multiple foods on an image?

### DISCUSSION THREAD

**Discussion by FlorianWoelki** [mrdbourke/nutrify#20] [Ideas]
I've noticed in a stream that some people were uploading food images that have multiple foods in the image. An example would be a bowl of fruits. The question is how to label them?

At the moment a user can insert just one food but also comma-separated foods. A really good improvement would be to have some kind of feature that lets you draw a bounding box around the food and label it from there.

What do you guys think about that?

**Comment by shivan-s** [comment]
Really good idea! Definitely one to add to to todos.

**Comment by mrdbourke** [comment]
Great idea, I've got a few things in mind here, many to do with Label Studio: https://labelstud.io

There will definitely be a data-labelling stream in the future (likely multiple).

My ideal: have a link from database to labelling tool so many people can just access images and start labelling, though we'd have to check the quality somehow 🤔
